arm-network/2016-09-01/swagger/network.json

```diff
@@ -4405,8 +4405,8 @@
           "description": "Health of backend server",
           "enum": [
             "Unknown",
-            "Up",
-            "Down",
+            "Healthy",
```

Comment:
If the service team is ok with this breaking change then should we merge this PR ?
/cc @sphibbs @markcowl